README.md

```diff
@@ -104,7 +118,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+validates :name :presence => true
```

Comment:
this question is actually referring to using an html validator online. the idea is that embedded ruby does not validate, so you would need to first grab the source code before running it